pkgs/by-name/al/alive2/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  clangStdenv,
+  fetchFromGitHub,
+  re2c,
+  z3,
+  hiredis,
+  llvm_18,
+  cmake,
+  ninja,
+}:
+
+clangStdenv.mkDerivation (finalAttrs: {
+  pname = "alive2";
+  version = "0-unstable-2024-09-23";
+
+  src = fetchFromGitHub {
+    owner = "AliveToolkit";
+    repo = "alive2";
+    rev = "05a964284056b38a6dc1f807e7acad64a0308328";
+    sha256 = "sha256-okKKUU7WLXLD9Hvsfoz+1HQWoyQ/bqRpBk5ogr7kSJA=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+  buildInputs = [
+    re2c
+    z3
+    hiredis
+    llvm_18
+    ninja
+  ];
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-fail 'find_package(Git REQUIRED)' "" \
```

```suggestion
      --replace-fail 'find_package(Git REQUIRED)' ""
```

And does this really work? Should we replace other commands?

---

Done. Well yeah it build and works fine for me locally.

---

Well I am not sure if we can replace the build commands with something shorter.